src/core/components/AMInstallButton/index.js

```diff
@@ -308,7 +310,7 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
     const buttonProps: ButtonProps = {
       buttonType: 'action',
       className: 'AMInstallButton-button',
-      disabled: buttonIsDisabled,
+      disabled: buttonIsDisabled || !installURL,
```

Comment:
Why not adding this condition to the `buttonIsDisabled` above?

Reply:
When I first tried to do that, it seemed to be getting unwieldy with the ternary, but I took another look and reversed the logic and now it seems okay to just add it into `buttonIsDisabled`, so I've done that.